src/app/directives/fl-container.js

```diff
@@ -57,7 +57,11 @@ export default function () {
 
       render() {
         this.container.removeGaps();
-        this.$element.css('height', this.mapper.height2px(this.container.getMaxHeight()) + 100)
+        if (this.isEditable) {
+          this.$element.css('height', this.mapper.height2px(this.container.getMaxHeight()) + 100);
```

Comment:
Instead of repeating - can you have

`this.$element.css('height', this.mapper.height2px(this.container.getMaxHeight()) + (this.isEditable? 100 : 0));`